lib/procfile.js

```diff
@@ -4,59 +4,84 @@ var updateCpu = require('./helpers/cpu')
 var parallel = require('./helpers/parallel')
 var history = require('./history')
 var cpuInfo = null
+var SIZE = 2048 // if the stat file is bigger then this I'll buy you a drink
```

Comment:
wtf? 😅
I've not looked at the code but cannot you just keep reading until EOF?

Reply:
was lazy :D

Reply:
laziness = bug
ahah 😩

Reply:
I bet you can't find a stat file bigger then this xD, I'm going to change it ;)